### DISCUSSION TITLE
in Orchestration, does not support zipkin?

### DISCUSSION THREAD

**Discussion by mrzxy** [apache/apisix#6065] [Q&A]
![image](https://user-images.githubusercontent.com/11501927/148747627-9cfa4e13-29a6-415f-b3a1-41415ceb6ab1.png)

**Comment by juzhiyuan** [comment]
Hi @mrzxy, may I know more about your "doesn't support"? Also, there has one V3 proposal, you could comment with your other focused features.

https://docs.qq.com/doc/DTkdBcG5IT3V6VER3
